crates/solvers/src/run.rs

```diff
@@ -18,19 +18,26 @@ pub async fn run(
     crate::boundary::initialize_tracing(&args.log);
     tracing::info!("running solver engine with {args:#?}");
 
-    let solver: Arc<dyn Solver> = match args.command {
+    let solver: Arc<dyn solver::Solver> = match args.command {
         cli::Command::Baseline { config } => {
             let baseline = config::baseline::file::load(&config).await;
-            Arc::new(baseline::Baseline {
+            Arc::new(solver::baseline::Baseline {
                 weth: baseline.weth,
                 base_tokens: baseline.base_tokens.into_iter().collect(),
                 max_hops: baseline.max_hops,
             })
         }
-        cli::Command::Naive => Arc::new(naive::Naive),
+        cli::Command::Naive => Arc::new(solver::naive::Naive),
         cli::Command::Legacy { config } => {
             let config = config::legacy::file::load(&config).await;
-            Arc::new(legacy::Legacy::new(config))
+            Arc::new(solver::legacy::Legacy::new(config))
```

Comment:
Would be nice to have solver re-exports to avoid naming saturation.
```suggestion
            Arc::new(solver::Legacy::new(config))
```

Reply:
328229745ad00f874faa31016511e2ae91e8a916